docs/v1/use-cases.md

```diff
@@ -3,6 +3,13 @@
 * TODO: scrub doc post glossary
 * TODO: add CF SB API as the lingua franca for blackbox services
 
+## Personas
+
+* User
+* Developer
+* Broker Operator
+* Service Provider
```

Comment:
For blackbox I would suggest that we don't draw a distinction between the person who manages the service broker and the person who owns the backend service - all we see is the service broker. Whether we call that "Broker Operator" or "Service Provider"....either works but Service Provider is the term I'm more used to.